sql/core/src/test/scala/org/apache/spark/sql/streaming/TriggerAvailableNowSuite.scala

```diff
@@ -128,7 +128,7 @@ class TriggerAvailableNowSuite extends FileStreamSourceTest {
           .option("maxFilesPerTrigger", 1)
           .text(src.getCanonicalPath)
 
-        val df2 = testSource.toDF
+        val df2 = testSource.toDF.selectExpr("cast(value as string)")
```

Comment:
explicit cast as string so that we don't need to concern the type coercion result of `long union string`